test/terraform/aks/main.tf

```diff
@@ -14,6 +14,8 @@ resource "azurerm_resource_group" "default" {
   count    = var.cluster_count
   name     = "consul-k8s-${random_id.suffix[count.index].dec}"
   location = var.location
+
+  tags = var.tags
```

Comment:
```suggestion
  tags = var.tags
```

Reply:
This was just a super minor suggestion but looks like you're doing this same thing elsewhere, no need to change it.

Reply:
yeah, this was intentional.